packages/canvas/canvas-text/src/Text.js

```diff
@@ -0,0 +1,23 @@
+import { Text } from '@pixi/text';
+import { Sprite } from '@pixi/sprite';
+
+/**
+ * Renders the object using the Canvas renderer
+ *
+ * @method _renderCanvas
+ * @memberof PIXI.Text#
+ * @private
+ * @param {PIXI.CanvasRenderer} renderer - The renderer
+ */
+Text.prototype._renderCanvas = function _renderCanvas(renderer) // eslint-disable-line no-unused-vars
```

Comment:
I don't think this comment is needed anymore. Render is used here.
```suggestion
Text.prototype._renderCanvas = function _renderCanvas(renderer)
```